central/main.go

```diff
@@ -683,10 +685,22 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 		})
 		customRoutes = append(customRoutes, routes.CustomRoute{
 			Route:         "/api/export/csv/node/cve",
-			Authorizer:    user.With(permissions.View(resources.Image), permissions.View(resources.Deployment), permissions.View(resources.Node)),
+			Authorizer:    user.With(permissions.View(resources.Node)),
 			ServerHandler: nodeCveCsv.NodeCVECSVHandler(),
 			Compression:   true,
 		})
+		customRoutes = append(customRoutes, routes.CustomRoute{
+			Route:         "/api/export/csv/image/cve",
+			Authorizer:    user.With(permissions.View(resources.Image), permissions.View(resources.Deployment)),
```

Comment:
@md2119 Are the permissions sufficient/correct for each new endpoint?

Reply:
No. You need to add deployment perm for image cves since the report is pulling deployment data as well. The rest looks good.

Reply:
@md2119 Deployment perm is added for image cves.

Reply:
Ah, not sure how I missed that!